plenum/server/node.py

```diff
@@ -1098,6 +1107,8 @@ def reset(self):
             self.metrics.add_event(MetricsName.LOOPER_RUN_TIME_SPENT, time.perf_counter() - self.last_prod_started)
         self.last_prod_started = time.perf_counter()
 
+        self.quota_control.set_request_queue_len(len(self.requests))
```

Comment:
There is a possible issue here:
As of now we have a problem (INDY-1248) that a request is not removed from the `requests` queue if at least one instance hasn't yet ordered it. If one of the instances doesn't have a Primary, then the requests queue will grow even if everything is totally fine on master and other instances.